### DISCUSSION TITLE
Yarn v4 timeframe?

### DISCUSSION THREAD

**Discussion by rbirkgit** [yarnpkg/berry#4895] [General]
I am curious what's the general time frame for v4 release? I cannot find any info about it.

**Comment by jj811208** [comment]
I saw in [this presentation](https://portal.gitnation.org/contents/yarn-4-modern-package-management) that yarn4 was originally targeting an October 2022 release (not a promise).

![yarn history](https://user-images.githubusercontent.com/25342059/192133772-7fdd854e-c540-4157-8d7b-df0ead5ff7b8.png)

But from this [issue](https://github.com/yarnpkg/berry/issues/3591), there is still much work to be done, so yarn4 should not have a definite release date yet

**Reply by arcanis** [reply]
Indeed - I've just started work on the website (which I'd like to release at the same time as the v4), and there are a couple of breaking changes I want to complete as well (yarn version, configuration merging); so it may take a couple more months.

That said, what's in master is stable, and I'd recommend you to try it. The only notable difference with stable is that we reserve the right to land a couple more breaking changes in future RCs, but in terms of stability it's almost always better to use RCs than stable.

**Reply by jj811208** [reply]
By the way, if anyone wants to try v4 RC, just run `yarn set version canary` in the project directory😀.

**Comment by rbirkgit** [comment]
Thanks! Looking forward using it when it comes out.
